Source/Core/Gen3/Searchers/WildSearcher3.cpp

```diff
@@ -22,9 +22,10 @@
 #include <Core/RNG/LCRNG.hpp>
 #include <Core/Util/EncounterSlot.hpp>
 
-WildSearcher3::WildSearcher3(u16 tid, u16 sid, u8 genderRatio, Method method, const StateFilter &filter) :
+WildSearcher3::WildSearcher3(u16 tid, u16 sid, u8 genderRatio, Method method, const StateFilter &filter, bool isRSEVersion) :
```

Comment:
Why are of any the changes in this file necessary? I don't account for method so why would whether or not it is safari zone matter. Some of the methods vary in rng calls as well.

Reply:
Fixed!

Reply:
Safari zone has got a different generation, so they're needed to match Generator results in case you are searching a spread for RSE safari zone